### ISSUE TITLE
Display is not implemented for Option<T>

### ISSUE THREAD

**Issue by omid** [jprochazk/garde#55]
I tried to integrate garde in our project. But I got this error:

```
    = help: the trait `std::fmt::Display` is not implemented for `std::option::Option<i32>`
    = note: in format strings you may be able to use `{:?}` (or {:#?} for pretty-print) instead
 note: required by a bound in `garde::rules::range::apply`
   --> /home/developer/registry/src/index.crates.io-6f17d22bba15001f/garde-0.12.0/src/rules/range.rs:19:26
    |
 19 | pub fn apply<T: Bounds + Display>(v: &T, (min, max): (&T::Size, &T::Size)) -> Result<(), Error> {
    |                          ^^^^^^^ required by this bound in `apply`
    = note: this error originates in the derive macro `Validate` (in Nightly builds, run with -Z macro-backtrace for more info)
```

The garde is `#[garde(range(min = 1))]` on a field with `Option<i32>` type.

**Comment by jprochazk** [comment]
Fixed and released in [`v0.13`](https://github.com/jprochazk/garde/releases/tag/v0.13.0)

**Comment by omid** [comment]
Wow, thanks
That was the fastest fix and release in the history ❤️
